pr2eus/make-pr2-model-file.l

```diff
@@ -1,11 +1,18 @@
-(defun make-pr2-model-file (&key (output-directory (ros::rospack-find "pr2eus")))
-  (let ((robot "pr2") s
-        fname_urdf fname_collada fname_yaml fname_lisp
-	valid-cameras camera-paths)
-    (unless (setq s (ros::get-param "/robot_description"))
-      (ros::ros-error "could not load model file from /robot_description~%")
-      (return-from make-pr2-model-file))
+;; re-define until https://github.com/jsk-ros-pkg/jsk_roseus/pull/629 is merged
+(defun ros::list-param ()
+  "Get the list of all the parameters in the server"
+  (let ((s (piped-fork "rosparam list")) tmp ret)
+    (while (setq tmp (read s nil))
+      (push (format nil "~A" tmp) ret))
+    ret))
+
+(defun get-pr2-names ()
+  ;; get list of rosparam
+  (reverse (mapcan #'(lambda (x) (if (substringp "robot_description" x) (list (string-left-trim "/" (string-right-trim "/robot_description" x)))))
```

Comment:
This usage of `string-right-trim` (and also `string-left-trim`?) seems dangerous.
See https://github.com/euslisp/EusLisp/issues/336.
For example, when `x` is `/pr/robot_description`, the output of `(string-left-trim "/" (string-right-trim "/robot_description" x))` is `p`.

Reply:
Seems resolved in the latest commit.